common/src/main/webapp/less/style.less

```diff
@@ -2243,3 +2244,25 @@ th.sticky-header,
 .ui.form .fields .field .ui.icon.input input {
   width: auto;
 }
+
+.ui.form .field.field input:-webkit-autofill,
+.ui.form .field.field input:-webkit-autofill:hover,
+.ui.form .field.field input:-webkit-autofill:focus,
+.ui.form .field.field input:-webkit-autofill:active,
+.ui.input input:-webkit-autofill,
+.ui.input input:-webkit-autofill:hover,
+.ui.input input:-webkit-autofill:focus,
+.ui.input input:-webkit-autofill:active,
+.ui.form input:-webkit-autofill,
+.ui.form input:-webkit-autofill:hover,
+.ui.form input:-webkit-autofill:focus,
+.ui.form input:-webkit-autofill:active,
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus,
+input:-webkit-autofill:active
```

Comment:
Does this only afect safari?

Reply:
No, it shouldn't, actually the video was done using chrome.

Reply:
Is this available as a standard field. In theory we should be able to generate the browser specific suffixes via postcss